src/Compilers/Extension/project.json

```diff
@@ -1,8 +1,12 @@
 ﻿{
   "dependencies": {
     "EnvDTE": "8.0.0",
-    "Microsoft.VisualStudio.Shell.14.0": "14.3.25407",
-    "Microsoft.VisualStudio.Shell.Interop.10.0": "10.0.30319"
+    "Microsoft.VisualStudio.Shell.15.0": "15.0.26014-alpha",
```

Comment:
There is an 'official' RC3 package we should probably be using instead: https://www.nuget.org/packages/Microsoft.VisualStudio.Shell.15.0/15.0.26109-RC3